src/_nebari/subcommands/init.py

```diff
@@ -484,7 +484,7 @@ def init(
             "-p",
             callback=typer_validate_regex(
                 schema.project_name_regex,
-                "Project name must (1) consist of only letters, numbers, hyphens, and underscores, (2) begin and end with a letter, and (3) contain between 3 and 32 characters.",
+                "Project name must (1) consist of only letters, numbers, hyphens, and underscores, (2) begin and end with a letter, and (3) contain between 3 and 16 characters.",
```

Comment:
When/how was the decision made to enforce a 16 character project name limit @iameskild ?  I'm curious because when I was working on naming conventions earlier in this release (mostly in https://github.com/nebari-dev/nebari/pull/2003/files), I did not end up coming to the conclusion that this was necessary (I left it at 32)

Reply:
This was the result of this discussion https://github.com/nebari-dev/nebari/issues/2037